code/style.css

```diff
@@ -1,4 +1,201 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif
 }
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+.header {
+  font-size: 30px;
+  font-weight: bold;
+  float: none;
+  color: black;
+  text-transform: initial;
+  letter-spacing: 5px;
+}
+
+nav {
+  width: 100%;
+  height: 75px;
+  line-height: 75px;
+  padding: 0px 100px;
+  background: lightgray
+}
+
+nav ul {
+  float: right;
+}
+
+nav li {
+  display: inline-block;
+  list-style: none;
+}
+
+nav li a {
+  font-size: 18px;
+  padding: 0px 30px;
+  color: #100e0d;
+  text-decoration: none;
+  display: flex;
+}
+
+nav li a:hover {
+  color: gray;
+}
+
+.main-news {
+  display: flex;
+  width: 100%;
+  background-color: #f6f6f6;
+}
+
+.big-image {
+  justify-content: center;
+  width: 50%;
+  margin: auto;
+}
+
+.main-text {
+  justify-content: center;
+  width: 50%;
+  text-align: left;
+  margin: auto;
+  padding: 20px;
+}
+
+h1 {
+  font-size: 22 px;
+  text-align: center;
+  padding: 40px;
+  color: black;
+}
+
+h2 {
+  font-size: 16px;
+  text-align: left;
+  padding: 10px;
+  color: black
+}
+
+.more-news {
+  width: 100%;
+  background: white;
+  min-height: 30vh;
+  margin-bottom: 20px;
+}
+
+.box {
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  min-height: 100px;
+}
+
+.card {
+  height: 250px;
+  padding: 15px 15px;
+  border-radius: 10px;
+  margin: auto;
+  background-color: lightgray;
+  position: relative;
+  overflow: hidden;
+  text-align: left;
+  border: 3px solid
+}
+
+.card-image {
+  height: 150px;
+  width: 200px;
+  margin: auto;
+  padding: 10px 5px;
+}
+
+footer {
+  background-color: lightgray;
+  color: #100e0d;
+  padding: 20px;
+  font-size: 14px;
+  color: black;
+  text-transform: initial;
+  letter-spacing: 2px;
+}
+
+@media only screen and (max-width: 1024px) {
+  .navigation ul {
+    display: none;
+  }
+
+  .main-news {
+    flex-wrap: wrap;
+  }
+
+
+  .main-text {
+    padding: 10px;
+  }
+
+  .big-image {
+    margin-top: 20px;
+  }
+
+  h1 {
+    font-size: 24px;
+  }
+
+  .main-text p {
+    font-size: 14px;
+  }
+
+  h2 {
+    font-size: 24px;
+    margin: 20px auto;
+  }
+
+  body {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .more-news {
+    min-height: 20vh;
+    margin-bottom: 20px;
+  }
+
+  .box {
+    grid-template-columns: repeat(3, 30%);
+  }
+
+  .card {
+    margin-bottom: 10px;
+  }
+
+  nav li a {
+    font-size: 14px;
+    padding: 0px 10px;
+  }
+}
+
+@media only screen and (max-width: 1600px) {
+  .box {
+    grid-template-columns: repeat((4, 20%));
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .box {
+    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
```

Comment:
we want to display 2 columns on tablets, so it's better to remove this rule to have full control of our layout. Try using simply grid-template-columns: repeat(2, 1fr)